src/utils/dates.js

```diff
@@ -0,0 +1,31 @@
+
+// () : Array<Number>
+export const MONTH_NAMES = (
+  'January February March April May June July August September October November December'
```

Comment:
Good point, we can make month names a prop and default to english? What do you think?